portfolio/src/main/java/com/google/sps/translation/TranslationProvider.java

```diff
@@ -26,14 +26,15 @@
 /** Class that allows to translate. */
 public class TranslationProvider {
   public static final String DEFAULT_TARGET_LANGUAGE = "en";
+  private static final String PROJECT_ID_ENVIRONMENT_VARIABLE = "GOOGLE_CLOUD_PROJECT";
   private Translate translate;
   private Set<String> supportedLanguageCodes;
 
   public TranslationProvider() {
     this.translate = 
       TranslateOptions.newBuilder()
-        .setProjectId("gioda-step-2020")
-        .setQuotaProjectId("gioda-step-2020")
+        .setProjectId(System.getenv(PROJECT_ID_ENVIRONMENT_VARIABLE))
```

Comment:
Since you're using this value in 2 places, you should read it from env just once, store in some local variable, and use the variable later on

Reply:
Thanks!